data-prepper-plugins/aws-plugin-api/build.gradle

```diff
@@ -12,7 +13,7 @@ jacocoTestCoverageVerification {
     violationRules {
         rule {
             limit {
-                minimum = 1.0
+                minimum = 0.95
```

Comment:
It would be very good to keep 100% coverage. Is there a certain line you are having difficulty reaching?

Reply:
Yes this line is difficult to reach (https://github.com/graytaylor0/data-prepper/blob/5f6fbaabc0ff4bf682073914cb99f5872d19a3c5/data-prepper-plugins/aws-plugin-api/src/main/java/org/opensearch/dataprepper/aws/api/AwsRequestSigningApache4Interceptor.java#L181)

Reply:
Because I can't mock the response of context.getAttribute to return an invalid HttpHost, and I can't mock it to throw either since that's a checked exception